L1Trigger/L1TMuonEndCap/interface/bdt/Event.h

```diff
@@ -27,8 +27,8 @@ struct Event
     int Quality;
```

Comment:
No changes to this file except spaces ... will probably make the review easier if we just leave it untouched.